File: java/Disk.java
public class Disk
{
    private String type = "";
    private String capacity = "";
    private int price = 0;

    Disk(){}

    Disk(String type, String capacity, int price)
    {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    public void setType(String type)
    {
        this.type = type;
    }

    public String getType()
    {
        return this.type;
    }

    public void setCapacity(String capacity)
    {
        this.capacity = capacity;
    }

    public String getCapacity()
    {
        return this.capacity;
    }

    public void setPrice(int price)
    {
        this.price = price;
    }

    public int getPrice()
    {
        return this.price;
    }

    public void showDisk()
    {
        System.out.println("Disk Type       : " + this.getType());
        System.out.println("Disk Capacity   : " + this.getCapacity());
        System.out.println("Disk Price      : " + this.getPrice());
    }
}

File: cpp/Disk.cpp
#include <iostream>
using namespace std;

class Disk
{
    private:
        string type = "";
        string capacity = "";
        int price = 0;

    public:
        Disk(){}

        Disk(string type, string capacity, int price)
        {
            this->type = type;
            this->capacity = capacity;
            this->price = price;
        }

        void setType(string type)
        {
            this->type = type;
        }

        string getType()
        {
            return this->type;
        }

        void setCapacity(string capacity)
        {
            this->capacity = capacity;
        }

        string getCapacity()
        {
            return this->capacity;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        int getPrice()
        {
            return this->price;
        }

        void showDisk()
        {
            cout << "Disk Type       : " << this->getType() << endl;
            cout << "Disk Capacity   : " << this->getCapacity() << endl;
            cout << "Disk Price      : " << this->getPrice() << endl;
        }

        ~Disk(){}
};